test/genegraph/transform/jsonld/common_test.clj

```diff
@@ -48,8 +48,7 @@
                            "@type" "http://example.org/Type1"
                            "field1" true
                            "field2" 25
-                           "field3" -100
-                           "field4" nil})
+                           "field3" -100})
```

Comment:
`nil` values are no longer included in output. This is consistent with the JSON-LD 1.1 specification.